common/CMakeLists.txt

```diff
@@ -45,12 +45,19 @@ set (SRC_LIST
     src/MachSyscall.C
   )
 
+set (C_SRC_LIST
+)
+
 if (PLATFORM MATCHES freebsd)
   set (SRC_LIST ${SRC_LIST} 
     src/timing-freebsd.C
     src/freebsdKludges.C
     src/addrtranslate-sysv.C
     src/addrtranslate-freebsd.C
+    src/symbolDemangleWithCache.C
+  )
+  set (C_SRC_LIST ${C_SRC_LIST}
+    src/symbolDemangle.c
```

Comment:
Could you remind me why we had to put this in a C-lang file by itself instead of in symbolDemangleWithCache?